app/controllers/submitted_proposals_controller.rb

```diff
@@ -38,8 +38,7 @@ def send_emails
   private
 
   def query_params?
-    (params.keys & %i[firstname lastname subject_area keywords workshop_year
-                      proposal_type]).any?
+    params.values.any?
```

Comment:
Please merge the development branch into your local branch. This bug was fixed @ https://github.com/birs-math/proposals/commit/8379867338685d15c323deefb97636135c36c2d6